app/services/container_service_mixin.rb

```diff
@@ -1,5 +1,6 @@
 module ContainerServiceMixin
-  CPU_USAGE_PRECISION = 2 # 2 decimal points
+  GRAPH_USAGE_PRECISION = 2 # 2 decimal points
+  DISPLAY_PRECISION = 0 # 2 decimal points
```

Comment:
0/2?

Reply:
Just a thought, would this read clearer if we would not use variables?
cc @cben

Reply:
At quick glance (didn't read whole PR) I think the consts help.  However I'm not sure what "graph usage" means.

Reply:
OK so vars it is. for GRAPH_USAGE_PRECISION I suggested GRAPH_PRECISION below because it makes sense to have one precision for all graphs

Reply:
@zeari what should be the value of DISPLAY_PRECISION here?